src/Raven.Server/Documents/Indexes/Sharding/Persistence/Lucene/ShardedLuceneIndexReadOperation.cs

```diff
@@ -15,6 +15,8 @@ namespace Raven.Server.Documents.Indexes.Sharding.Persistence.Lucene;
 
 public sealed class ShardedLuceneIndexReadOperation : LuceneIndexReadOperation
 {
+    protected override Sort SortByFieldScore => throw new NotSupportedInShardingException("Ordering by score is not supported in sharding. You received this exception because your index has boosting, and we attempted to sort the results since the configuration `Indexing.OrderByScoreAutomaticallyWhenBoostingIsInvolved` is enabled.");
```

Comment:
Instead of hard coding the configuration name you can use `RavenConfiguration.GetKey()`

Reply:
done